### ISSUE TITLE
Assistant Has hardcoded file store

### ISSUE THREAD

**Issue by dahifi** [culture-dao/chainlit-template#30]
```
        if vector_store_id is None:
            # get "Default Datastore"
            vector_store_id = 'vs_ZpE5J5qh5KMRMrwXkzsAxobM'
```

**Comment by dahifi** [comment]
Fixed in ddd84ae5510088a8e1cf67f09f8a9d38ef0563f5, 

We can figure something fancy when we get into the vector handlers. We might not need to worry about this function since we're probably not going to be swapping datastores around in the app itself, but we might be able to specify this in the config somewhere. This id should be dynamic anyways.

**Comment by Lauren1066** [comment]
Merged https://github.com/culture-dao/chainlit-template/pull/33
